src/main/webapp/app/core/interceptor/artemis-version.interceptor.ts

```diff
@@ -3,7 +3,7 @@ import { HttpEvent, HttpHandler, HttpInterceptor, HttpRequest, HttpResponse } fr
 import { Observable, Subject } from 'rxjs';
 import { tap, throttleTime } from 'rxjs/operators';
 import { ARTEMIS_VERSION_HEADER, VERSION } from 'app/app.constants';
-import { AlertService } from 'app/core/util/alert.service';
+import { Alert, AlertCreationProperties, AlertService } from 'app/core/util/alert.service';
```

Comment:
```suggestion
import { AlertService } from 'app/core/util/alert.service';
```
The imports are probably unused again as there are no other changes in this file.

Reply:
🙄 Damn. Will let IntelliJ perform another import optimizer run :D